src/js/components/Notification.js

```diff
@@ -48,12 +49,15 @@ export default class Notification extends Component {
     let progress;
     if (this.props.percentComplete || 0 === this.props.percentComplete) {
       progress = (
-        <Meter units="%"
-          series={[{
-            value: this.props.percentComplete,
-            label: '',
-            colorIndex: 'light-1'
-          }]} />
+        <span>
```

Comment:
I think this should be a `<Box direction="row" align="center">` to align the Meter and the Value.